README.md

```diff
@@ -389,6 +389,7 @@ miette::set_hook(Box::new(|_| {
         .terminal_links(true)
         .unicode(false)
         .context_lines(3)
+        .with_tab_width(4)
```

Comment:
this should be just `.tab_width`

Reply:
Done cb073dd3e773e5ac39db7d3ac811fcbf6c53e82d